cdc/sinkv2/event/row_change_event_appender.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package event
+
+import "github.com/pingcap/tiflow/cdc/model"
+
+// Assert Appender[E TableEvent] implementation
+var _ Appender[*model.RowChangedEvent] = (*RowChangeEventAppender)(nil)
+
+// RowChangeEventAppender is the builder for RowChangedEvent.
+type RowChangeEventAppender struct{}
```

Comment:
why is this struct needed ?

Reply:
Because golang doesn't support type function generics, otherwise we would just support a function.

Reply:
![image](https://user-images.githubusercontent.com/29879298/175447710-731102ea-1040-4e72-abc7-c145dcdf6626.png)